advanced/audio.md

```diff
@@ -32,10 +32,24 @@ If the incorrect version is being used then it is possible that some announcemen
 
 A sound pack is available from within Companion.
```

Comment:
how about "A sound pack is available from withing companion as part of the SD Card contents."